client/v2/client.go

```diff
@@ -119,6 +123,49 @@ func NewHTTPClient(conf HTTPConfig) (Client, error) {
 	}, nil
 }
 
+// Pings cluster
+func (c *client) Ping(timeout time.Duration) (time.Duration, string, error) {
```

Comment:
I don't mind making this change and in fact I already have it in my tree, but I went to confirm that `golint` was happy and it seems it doesn't actually care about the plural form. Are there some options I need to use to get it to be strict about that?

    $ golint
    client.go:24:6: exported type HTTPConfig should have comment or be unexported
    client.go:50:6: exported type UDPConfig should have comment or be unexported
    client.go:60:6: exported type BatchPointsConfig should have comment or be unexported
    client.go:91:1: comment on exported function NewHTTPClient should be of the form "NewHTTPClient ..."
    client.go:321:6: exported type Point should have comment or be unexported
    client.go:364:1: comment on exported method Point.Tags should be of the form "Tags ..."

Let me know if I should push my updated comments for consistency. I can also fix the other comments in a separate PR.

Thanks.